airbyte-integrations/connectors/source-surveymonkey/source_surveymonkey/streams.py

```diff
@@ -132,11 +132,14 @@ def path(self, **kwargs) -> str:
         return "surveys"
 
     def parse_response(self, response: requests.Response, stream_state: Mapping[str, Any], **kwargs) -> Iterable[Mapping]:
+        params = super().request_params(stream_state=stream_state, **kwargs)
+        survey_ids = params.get("survey_ids", [])
```

Comment:
I have missed flow of this parameter `survey_ids` from config.json to this point?

Reply:
i'm not sure what's required here

Reply:
`survey_ids` is not request_params, it's param which passed from `config.json`
You need to update `source.py` and pass to all streams `survey_ids` param similar way like we did for `start_date`